lbfgs_linear_clf_spirograph.py

```diff
@@ -0,0 +1,151 @@
+'''This script tunes the L2 reg weight of the final classifier.'''
```

Comment:
What's the difference between this script and `lbfgs_linear_clf_spirograph_cp.py`? Do we need both?